packages/utils/src/f1-field.ts

```diff
@@ -7,6 +7,8 @@ import * as scalar from "./scalar"
  * and inversion, all performed modulo the field's order. It's designed to work with bigints,
  * supporting large numbers for cryptographic purposes and other applications requiring
  * modular arithmetic.
+ * Note that even if the outputs of the functions are always within the field,
```

Comment:
this isn't true. outputs are within field range if inputs are in range.

Reply:
You're right :D let me update it

Reply:
Please, check it again